### PR TITLE
[Python] Add python-shell-send-line function

### PR DESCRIPTION
**Pull request by beyondpie**
Add the python-shell-send-line function as in the other language layer, such as ESS. And bind it also as the same key.

### CODE REVIEW

**Comment by beyondpie** [comment]
I am not familiar with Elisp. So I just use the ESS layer and python layer as a reference, and add this function. 
- should I use `let` or `let*`?  I use `let*` as used in ESS. 
- for the `interactive` part, should I add an "r" argument? I see most of the functions in the Python layer, they use this. But in ESS, not. I choose not to add this.

File: layers/+lang/python/packages.el [hunk 1]
```diff
@@ -354,7 +354,8 @@
         "sf" 'spacemacs/python-shell-send-defun
         "si" 'spacemacs/python-start-or-switch-repl
         "sR" 'spacemacs/python-shell-send-region-switch
-        "sr" 'spacemacs/python-shell-send-region)
+        "sr" 'spacemacs/python-shell-send-region
+				"sl" 'spacemacs/python-shell-send-line)
```
Comment (Compro-Prasad) [inline layers/+lang/python/packages.el:358]: This line is over indented.

File: layers/+lang/python/funcs.el [hunk 2]
```diff
@@ -440,6 +440,15 @@ Bind formatter to '==' for LSP and '='for all other backends."
   (let ((python-mode-hook nil))
     (python-shell-send-region start end)))
 
+(defun spacemacs/python-shell-send-line ()
+	"Send the current line to shell"
+	(interactive)
+	(let* ((python-mode-hook nil)
+				 (start (point-at-bol))
```
Comment (Compro-Prasad) [inline layers/+lang/python/funcs.el:447]: This line is over indented.

File: layers/+lang/python/funcs.el [hunk 3]
```diff
@@ -440,6 +440,15 @@ Bind formatter to '==' for LSP and '='for all other backends."
...
+				 (end (point-at-eol)))
```
Comment (Compro-Prasad) [inline layers/+lang/python/funcs.el:448]: This line is over indented.

**Comment by Compro-Prasad** [comment]
`let*` would work but `let` would be better.

**Comment by beyondpie** [comment]
@Compro-Prasad 
Thanks for your comments! I've updated the files.

**Comment by Compro-Prasad** [comment]
Add some docs if possible. It would be helpful for starters.

**Comment by beyondpie** [comment]
> Add some docs if possible. It would be helpful for starters.

Done.

**Review by smile13241324 — APPROVED** [review]
Looks good, will be merged soon.

**Pull request merged**